runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -3304,3 +3304,82 @@ J9::ValuePropagation::innerConstrainAcall(TR::Node *node)
 
    return node;
    }
+
+TR_OpaqueClassBlock *
+J9::ValuePropagation::findLikelySubtype(TR_OpaqueClassBlock *klass)
+   {
+   if (!klass || TR::VPConstraint::isSpecialClass((uintptr_t)klass))
+      return NULL;
+
```

Comment:
Would you mind adding an early exit here to return null in non-SVM AOT? I'd like to make sure there are no hints in that case. In fact, maybe for the moment we should return null in all AOT compilations, since even in the SVM case we need to make sure that any dynamic type test we end up generating based on the hint will be relocated correctly

Reply:
Addressed in 4f8de01